incubator/baget/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "baget.fullname" . }}
+  labels:
+    app: {{ template "baget.name" . }}
```

Comment:
please use the new labels https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels